From 17af5afc701fd8d053db19418815ce4a6e28609f Mon Sep 17 00:00:00 2001
From: Nicolas Le Bayon <nicolas.le.bayon@foss.st.com>
Date: Fri, 22 Nov 2024 14:45:47 +0100
Subject: [PATCH 742/954] fix(stm32mp25-fdts): new DDR4 density comment for
 STM32MP257F-EV1 board

Describe density only per device (its width is defined by device_width).

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 4278b5ddbacfdf92f3ea4c1dd3d5564d8048eebc fix(stm32mp25-fdts): new DDR4 density comment for STM32MP257F-EV1 board]
Signed-off-by: Nicolas Le Bayon <nicolas.le.bayon@foss.st.com>
Change-Id: I91ba9142b5582734cfcb4ac2209434a2fcf47591
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/420375
Tested-by: Nicolas LE BAYON <nicolas.le.bayon@st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Reviewed-by: Nicolas LE BAYON <nicolas.le.bayon@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 fdts/stm32mp25-ddr4-2x16Gbits-2x16bits-1200MHz.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdts/stm32mp25-ddr4-2x16Gbits-2x16bits-1200MHz.dtsi b/fdts/stm32mp25-ddr4-2x16Gbits-2x16bits-1200MHz.dtsi
index 3d69448ad5..d0a6be7b56 100644
--- a/fdts/stm32mp25-ddr4-2x16Gbits-2x16bits-1200MHz.dtsi
+++ b/fdts/stm32mp25-ddr4-2x16Gbits-2x16bits-1200MHz.dtsi
@@ -13,7 +13,7 @@
  * Speed_Bin    Worse    from JEDEC
  * device_width 16       x16 by default
  * width        32       32: full width / 16: half width
- * density      16Gbits  (per 16bit device)
+ * density      16Gbits  (per device)
  * Addressing   RBC      row/bank interleaving
  * RDBI         No       Read DBI
  */
-- 
2.53.0

